apps/settings/locales/settings.en-US.properties

```diff
@@ -477,7 +477,7 @@ bluetooth-devices-in-area  = Devices in the area
 bluetooth-visible-to-all = Visible to all
 bluetooth-name-maxlength-alert = Your device name exceeds {{length}} characters. Please retry again.
 bluetooth-address = Bluetooth Address
-bluetooth-address-unavailable = Not available
+bluetooth-address-unavailable = Bluetooth has been disabled
```

Comment:
for this case, I think `disabled` is sufficient, so you can reuse the `disabled` identifier